recipe/meta.yaml

```
@@ -62,8 +65,12 @@ test:
     - setuptools
   commands:
     - f2py -h
+    # show SIMD features (some failures occur depending on presence/absence of e.g. AVX512)
+    # change to np.show_config() for v>=1.22.0, cf. numpy/numpy#19130
+    - python -c "import numpy; numpy._pytesttester._show_numpy_info()"
     {% set numproc = "-nauto" if aarch64 else "" %}
-    - pytest --pyargs numpy -k "not ({{ tests_to_skip }})" {{ numproc }} --durations=100 -v
+    - python -c "import numpy as np; np.test(verbose=2, durations=100, extra_argv=['-k', 'not ({{ tests_to_skip }})', '{{ numproc }}'])"          # [not (win and python_impl == "pypy")]
+    - pypy --jit off -c "import numpy as np; np.test(verbose=2, durations=100, extra_argv=['-k', 'not ({{ tests_to_skip }})', '{{ numproc }}'])"  # [win and python_impl == "pypy"]
```

Try a run with the JIT just to prove it causes the failures?
```suggestion
    - python -c "import numpy as np; np.test(verbose=2, durations=100, extra_argv=['-k', 'not ({{ tests_to_skip }})', '{{ numproc }}'])"          # [not (win and python_impl == "pypy")]
    - pypy -c "import numpy as np; np.test(verbose=2, durations=100, extra_argv=['-k', 'not ({{ tests_to_skip }})', '{{ numproc }}'])"  # [win and python_impl == "pypy"]
```